src/rulesets/reader.ts

```diff
@@ -97,6 +101,7 @@ const createRulesetProcessor = (
         if (extendedRuleset !== null) {
           mergeRules(rules, extendedRuleset.rules, parentSeverity);
           Object.assign(functions, extendedRuleset.functions);
+          mergeExceptions(baseUri, exceptions, extendedRuleset.exceptions);
```

Comment:
Food for thoughts, but I'm pretty sure we'll want to filter out certain exceptions.
For instance,
```json
{
  "file/#: ["foo-rule"],
  "file/#.bar": ["foo-rule"]
}
```
This could be expressed in a single exception `"file/#": ["foo-rule"]`.
We'll have fewer paths to query later if we narrow them down here.

Reply:
@P0lip Is that a use case so common that it should be included in that PR or should we wait until we get some feedback that maintaining exceptions is too tedious?